### PR TITLE
Creature kill corpse - For review - NOT ready for a merge

### PR DESCRIPTION
**Pull request by Lidefeath**

### CODE REVIEW

File: Source/ACE/Network/GameEvent/Events/GameEventDeathNotice.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿namespace ACE.Network.GameEvent.Events
```
Comment (fantoms) [inline Source/ACE/Network/GameEvent/Events/GameEventDeathNotice.cs]: This is also needed for player death notices, so this code would be useful to have as a separate PR, for Merging.

**Comment by Lidefeath** [comment]
As discussed with fantoms, I'll redo this as a proper commit.
